x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/source_selection/source_selection.tsx

```diff
@@ -31,11 +23,7 @@ import { getDataViewAndSavedSearch, isCcsIndexPattern } from '../../../../../uti
 
 const fixedPageSize: number = 8;
```

Comment:
Now that this is in its own page, this could be increased to `10`, or the Anomaly Detection one uses `20` I think.